README.md

```diff
@@ -342,6 +342,13 @@ Thanks goes to these wonderful people ([:hugs:](https://allcontributors.org/docs
                     <sub><b>Siddharth Patel</b></sub>
                   </a>
             </td>
+            <td align="center">
```

Comment:
You should actually add your own information instead, but since this is a Welcome to Open Source project, it does not matter too much. Nevertheless, you should add correct indention, provide a valid href value for the anchor, add a correct image source instead of a random website, add the attribute with="100px" and an accessible alt text to your image.